Automattic/wp-calypso
# [Coming Soon] Update "Private" copy to "Coming Soon"

#### Changes proposed in this Pull Request

This PR updates site status badge to say "Coming soon" for appropriate sites. More context available here: paObgF-Ui-p2

<img width="407" alt="Zrzut ekranu 2020-01-15 o 15 29 46" src="https://user-images.githubusercontent.com/205419/72441949-e3435f00-37ab-11ea-8d57-7e5ad4260cec.png">

#### Testing instructions

1. Apply D37570-code
1. Set one of your simple sites to private
1. Also set it to "Coming Soon" mode by setting `wpcom_coming_soon` option to 1
1. Confirm the badge says "Coming Soon"
1. Confirm badges for other, private sites still say "Private"
